Sources/Variable.swift

```diff
@@ -75,6 +75,24 @@ public struct Variable : Equatable, Resolvable {
       return bool
     }
 
+    func resolveCollection<T: Collection>(_ collection: T, bit: String) -> Any? {
```

Comment:
I'd actually move this out of the body of `resolve(`, I'm preparing another PR that adds swiftlint, and this will only worsen the situation. Maybe just a `private func` right after this function?